src/wp-includes/script-loader.php

```diff
@@ -1302,6 +1308,8 @@ function wp_default_scripts( $scripts ) {
 		$scripts->add( 'custom-background', "/wp-admin/js/custom-background$suffix.js", array( 'wp-color-picker', 'media-views' ), false, 1 );
 		$scripts->add( 'media-gallery', "/wp-admin/js/media-gallery$suffix.js", array( 'jquery' ), false, 1 );
 
+		$scripts->add( 'media-events', "/wp-admin/js/media-events$suffix.js", array( 'image-edit', 'jquery' ), false, 1 );
```

Comment:
Where is this being enqueued? I don't see any other references to 'media-events' in this PR.

Reply:
The diff is hidden because it's too big: https://github.com/WordPress/wordpress-develop/pull/551/files#diff-791daa8f324d2c9aabc1a122031f605b